### PR TITLE
Add SwaggerResponseAttribute.ContentTypes property

### PR DESCRIPTION
**Pull request by RehanSaeed**
Fixes https://github.com/domaindrivendev/Swashbuckle.AspNetCore/issues/1691

### CODE REVIEW

**Review by RehanSaeed — COMMENTED** [review]
@domaindrivendev Made changes as requested.

File: src/Swashbuckle.AspNetCore.Annotations/SwaggerResponseAttribute.cs [hunk 1]
```diff
@@ -15,9 +15,21 @@ public SwaggerResponseAttribute(int statusCode, string description = null, Type
             Description = description;
         }
 
+        public SwaggerResponseAttribute(int statusCode, string description = null, Type type = null, params string[] contentTypes)
```
[the remark this answers is not in the document]
Comment (domaindrivendev) [reply src/Swashbuckle.AspNetCore.Annotations/SwaggerResponseAttribute.cs:18]: That's why I'm suggesting a second constructor overload. For example:

```chsarp
// For the regular usage
public SwaggerResponseAttribute(int statuscode, string description = null, Type type = null)

// For the new usage scenario you're adding support for
public SwaggerResponseAttribute(int statuscode, string[] contentTypes, Type type, string description = null);
```

[the remark this answers is not in the document]
Comment (RehanSaeed) [reply src/Swashbuckle.AspNetCore.Annotations/SwaggerResponseAttribute.cs:18]: In your example, `contentTypes` stops using `params`, also the `type` parameters position gets swapped around, so if you want to switch between which constructor you want to use, you'd have to annoyingly manually move parameters around.

Would you be happy with three constructor parameters, with the first original one possibly being marked as Obsolete?

```
    public SwaggerResponseAttribute(int statusCode, string description = null, Type type = null)
            : base(type ?? typeof(void), statusCode)
        {
            Description = description;
        }

        public SwaggerResponseAttribute(int statusCode, Type type = null, string description = null)
            : base(type ?? typeof(void), statusCode)
        {
            Description = description;
        }

        public SwaggerResponseAttribute(int statusCode, Type type, string description = null, params string[] contentTypes)
            : base(type, statusCode)
        {
            Description = description;
            ContentTypes = contentTypes;
        }
```

[the remark this answers is not in the document]
Comment (RehanSaeed) [reply src/Swashbuckle.AspNetCore.Annotations/SwaggerResponseAttribute.cs:18]: @domaindrivendev Polite ping.

**Pull request merged**